lib/bolt/encryptors.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+require 'bolt/encryptors/pkcs7'
+
+module Bolt
+  class Encryptors
+    def initialize(config)
+      @config = config
+      @encryptors = { 'PKCS7' => Bolt::Encryptors::Pkcs7.new(config.boltdir.path, config.plugins['PKCS7']) }
+    end
+
+    def encode(raw, plugin)
+      coded = Base64.encode64(raw).strip
+      "ENC[#{plugin},#{coded}]"
+    end
+
+    def decode(code)
+      format = %r{\AENC\[(?<plugin>\w+),(?<encoded>[\w\s+-=/]+)\]\z}
+      match = format.match(code)
+
+      # TODO: should we error if the plugin doesn't match?
+      raise "Could not parse as an encrypted value: #{code}"
+
+      raw = Base64.decode64(match[:encoded])
+      [raw, match[:plugin]]
+    end
+
+    def execute(action, object, _options)
+      enc = @encryptors['PKCS7']
+      case action
+      # TODO: should this be createkeys for eyaml compatibility
+      when 'create-keys'
```

Comment:
Unless there's a compelling reason I'm missing I think `createkeys` everywhere makes sense. It makes it easier to search for all the places that's referred to, and seems standard, and we're already using it in most cases of `create*keys`.

Reply:
hmm I thought I fixed this, I'm not sure how the integration tests are passing

Reply:
ah this file is no longer used.